DMCompiler/DMStandard/UnsortedAdditions.dm

```diff
@@ -0,0 +1,161 @@
+﻿
+/proc/addtext(...) 
+/proc/bounds_dist(Ref, Target)
+/proc/copytext_char(T,Start=1,End=0)
+/proc/link(url)
+/proc/filter(type, parameter, ...)
+/proc/findlasttext_char(Haystack,Needle,Start=0,End=1)
+/proc/findtext_char(Haystack,Needle,Start=1,End=0)
+/proc/issaved(v)
+/proc/shell(command)
+/proc/run(File)
+/proc/ftp(File, Name)
+/proc/replacetext_char(Haystack,Needle,Replacement,Start=1,End=0)
+/proc/rgb2num(color, space)
+/proc/spantext_char(Haystack,Needles,Start=1)
+/proc/winget(player, control_id, params)
+/proc/winexists(player, control_id)
+/proc/winclone(player, window_name, clone_name)
+/proc/winshow(player, window, show=1)
+/proc/walk_rand(Ref,Lag=0,Speed=0)
+
+/atom
+	var/filters = list()
```

Comment:
I worry that stubbing vars like this is going to lead to silently broken code. I'd like to have some way of flagging vars like this as unimplemented, similar to what we have for procs right now. This was brought up in #368.